src/lib/devConsole.js

```diff
@@ -1015,18 +1015,11 @@ const getPublicEncryptionKey = async organizationCode => {
 	}
 };
 
-const getPresignedUrls = async (
-	organizationCode,
-	projectId,
-	workspaceId,
-	meshId,
-	startTime,
-	endTime,
-) => {
+const getLogsByRayId = async (organizationId, projectId, workspaceId, meshId, rayId) => {
 	const { accessToken, apiKey } = await getDevConsoleConfig();
 	const config = {
 		method: 'get',
-		url: `${SMS_BASE_URL}/organizations/${organizationCode}/projects/${projectId}/workspaces/${workspaceId}/meshes/${meshId}/logs?startDateTime=${startTime}&endDateTime=${endTime}&API_KEY=${apiKey}`,
+		url: `${SMS_BASE_URL}/organizations/${organizationId}/projects/${projectId}/workspaces/${workspaceId}/meshes/${meshId}/logs/${rayId}?API_KEY=${apiKey}`,
```

Comment:
Since we bypass Dev console and call SMS directly, shouldn't we be sending the `organizationCode` instead of `organizationId` ?

Reply:
yes , will change it to organizationCode